mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,266 @@
+
+```shell
+$ qsub run.sh
+{{jobid}}
+```
+
+This command returns a job identifier (*{{jobid}}*) on the HPC cluster. This is a unique identifier for the job which can be used to monitor and manage your job.
+
+!!! Warning
+ 
+    Note that the module commands only modify environment variables. For instance, running `module swap cluster/{{othercluster}}` will update your shell environment so that `qsub` submits a job to the {{othercluster}} cluster, 
+    but our active shell session is still running on the login node.
+    
+    It is important to understand that while `module` commands affect your session environment, they do ***not*** change where the commands your are running are being executed: they will still be run on the login node you are on.
+    
+    When you submit a job script however, the commands ***in*** the job script will be run on a workernode of the cluster the job was submitted to (like `{{othercluster}}`).
+
+For detailed information about `module` commands, read the [running batch jobs](running_batch_jobs.md) chapter.
+
+### Wait for job to be executed
+
+Your job is put into a queue before being executed, so it may take a while before it actually starts.
+(see [when will my job start?](running_batch_jobs.md#when-will-my-job-start) for scheduling policy)
+
+You can get an overview of the active jobs using the `qstat` command:
+<pre><code>$ qstat
+Job ID     Name             User            Time Use S Queue
+---------- ---------------- --------------- -------- - -------
+{{jobid}}     run.sh           {{userid}}        0:00:00  <b style="color:orange">Q</b> {{othercluster}}
+</code></pre> 
+
+Eventually, after entering `qstat` again you should see that your job has started running:
+<pre><code>$ qstat
+Job ID     Name             User            Time Use S Queue
+---------- ---------------- --------------- -------- - -------
+{{jobid}}     run.sh           {{userid}}        0:00:01  <b style="color:green">R</b> {{othercluster}}
+</code></pre> 
+
+If you don't see your job in the output of the `qstat` command anymore, your job has likely completed.
+
+Read [this section](running_batch_jobs.md#monitoring-and-managing-your-jobs) on how to interpret the output.
+
+### Inspect your results
+
+When your job finishes it generates 2 output files:
+
+- One for normal output messages (*stdout* output channel).
+- One for warning and error messages (*stderr* output channel).
+
+By default located in the directory where you issued `qsub`.
+
+{%- if site == 'Gent' %}
+
+!!! Info
+
+    For more information about the stdout and stderr output channels, see this [section](./only/gent/linux-tutorial/beyond_the_basics.md#inputoutput).
+
+{%- endif %}
+
+In our example when running <code>ls</code> in the current directory you should see 2 new files:
+ 
+- **run.sh.o{{jobid}}**, containing normal output messages produced by job {{jobid}};
+- **run.sh.e{{jobid}}**, containing error and warning messages produced by job {{jobid}}.
```

Comment:
```suggestion
- **run.sh.o{{jobid}}**, containing *normal output messages* produced by job {{jobid}};
- **run.sh.e{{jobid}}**, containing *errors and warnings* produced by job {{jobid}}.

Reply:
We should probably also add a warning box here that the example job ID `123456` should be replaced by the actual job ID of the submitted job?